### DISCUSSION TITLE
Set "Watching without audio" badge

### DISCUSSION THREAD

**Discussion by exstrim401** [Chatterino/chatterino2#3804] [Ideas] [answered]
Add ability to set "Watching without audio" and "Wathing without video" badges
![screenshot](https://user-images.githubusercontent.com/57351693/172228690-c921c4fa-e05f-440c-bde6-a0e329e2f3d2.png)

**Comment by Mm2PL** [comment, marked as answer]
Impossible. Ask twitch for an API for this.

**Comment by Felanbird** [comment]
As Mm2PL has mentioned, there is not, nor ever has been, an API allowing users to change their badges. 
I'm fairly sure there is no mention of badges anywhere within the Helix API, even to check what badge a user has enabled.
